crates/nu-protocol/src/value.rs

```diff
@@ -320,6 +320,27 @@ impl Value {
         }
     }
 
+    /// View the Value as a BigInt, if possible
+    pub fn as_bigint(&self) -> Result<BigInt, ShellError> {
+        match &self.value {
+            UntaggedValue::Primitive(Primitive::Filesize(fs)) => Ok(fs.clone()),
+            UntaggedValue::Primitive(Primitive::Duration(dur)) => Ok(dur.clone()),
+            UntaggedValue::Primitive(Primitive::Int(n)) => Ok(n.clone()),
+            _ => Err(ShellError::type_error("bigint", self.spanned_type_name())),
+        }
+    }
```

Comment:
Please don't add this one. Stripping the type away from the number is going to lead to many subtle bugs later on.

Instead, make sure the method and the type go together. `as_filesize`, `as_duration`, `as_int`, etc